lib/server/env.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const CWD = process.cwd();
+const fs = require('fs-extra');
+const path = require('path');
+
+const join = path.join;
+
+const languages_js = join(CWD, 'lanauages.js');
+const translation_js = join(CWD, 'translation.js');
```

Comment:
This needs to be `const translation_js = './translation.js`;` since this file is in the same directory as `env.js`, not as part of the current process' working directory.